scan

```diff
@@ -37,8 +37,13 @@ start_command = str.join(" ", sys.argv)
 # Generate a random UUID for the entire scan.
 scan_uuid = str(uuid.uuid4())
 
+# Whitelist of scanners that can use lambda
+# (facilitates running a mix of lambda / non-lambda scanners together)
+lambda_supported = ["pshtt", "sslyze"]
+lambda_mode = options.get("lambda", False)
```

Comment:
Instead of hardcoding the list here, could we add this as an attribute on the `pshtt` and `sslyze` scanner modules? Then any scanner could opt-in to advertising its lambda support without code changes in the central orchestration script.

Reply:
sure, we can do it that way -- will update

Reply:
updated!